### PR TITLE
Rename \ to \\ to build on 0.13

### LINKED ISSUE #2: Build failure on purs-0.13
```
Error found:
at src/CallByName/Syntax.purs:16:25 - 16:26 (line 16, column 25 - line 16, column 26)

  Unable to parse module:
  Unexpected token '\'


See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
or to contribute content related to this error.


* ERROR: Subcommand terminated with exit code 1
```

### PR DESCRIPTION
**Pull request by rnons**
Close #2.

### CODE REVIEW

File: test/Main.purs [hunk 1]
```diff
@@ -36,11 +36,11 @@ main = do
...
-  when false \do
+  when false \_ ->
     unsafeThrow "Too strict!"
 
-  unless true \do
+  unless true \_ ->
     unsafeThrow "Too strict!"
 
-  guard false \do
+  guard false \_ ->
```
Comment (natefaubion) [inline test/Main.purs]: Did you intend to add `\\` as an operator? The point of these tests is to test that the operator works, so by not using the operator, they no longer test anything ;).

Comment (rnons) [reply test/Main.purs]: Fixed, I though these were testing guard/when/unless

**Comment by natefaubion** [comment]
Do you mind updating the README?

**Comment by rnons** [comment]
Done.

**Comment by i-am-the-slime** [comment]
"We've saved **five** arduous characters" should probably become "four"

**Comment by rnons** [comment]
Fixed

**Review by natefaubion — APPROVED** [review]

**Comment by natefaubion** [comment]
Thanks!

**Pull request merged**